daisi/src/path_planning/pickup_station.cpp

```diff
@@ -164,9 +164,9 @@ void PickupStation::update(const std::string &agv_id) {
     // Pack to/from station IDS into first element of vector
     ns3::Vector pickup(agv.current_to.delivery_station, -1, -1);
     ns3::Vector delivery(info_.station_id, -1, -1);
-    cpps::Task task(
-        agv.current_to.uuid, pickup, delivery,
-        {100.0F, cpps::mrta::model::LoadCarrier(cpps::mrta::model::LoadCarrier::kEuroBox)});
+    cpps::Task task(agv.current_to.uuid, pickup, delivery,
+                    cpps::amr::AmrStaticAbility(
+                        cpps::amr::LoadCarrier(cpps::amr::LoadCarrier::kEuroBox), 100.0F));
```

Comment:
Does it make sense to define the 100.0F as a constant somewhere or use a default value for the variable in the constructor? (also used down in line 430 and in `daisi/src/path_planning/path_planning_manager.cpp`)

Reply:
I did not write the code. I just changed it such that it compiles.